maps/race_drought/race.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0"?>
 <map proto="1.3.0">
 <teams>
-    <team color="blue" max="1" max-overfill="1">Blue Car</team>
-    <team color="red" max="1" max-overfill="1">Red Car</team>
-    <team color="dark green" max="1" max-overfill="1">Green Car</team>
-    <team color="yellow" max="1" max-overfill="1">Yellow Car</team>
-    <team color="dark purple" max="1" max-overfill="1">Purple Car</team>
-    <team color="light purple" max="1" max-overfill="1">Pink Car</team>
-    <team color="gold" max="1" max-overfill="1">Orange Car</team>
-    <team color="green" max="1" max-overfill="1">Lime Car</team>
-    <team color="dark aqua" max="1" max-overfill="1">Cyan Car</team>
-    <team color="gray" max="1" max-overfill="1">Gray Car</team>
-    <team color="dark blue" max="1" max-overfill="1">Navy Car</team>
-    <team color="black" max="1" max-overfill="1">Black Car</team>
+    <team id="blue" color="blue" max="1" max-overfill="1">Blue Car</team>
```

Comment:
Not sure if ids are needed, the map proto doesn't require them

Reply:
Are you sure? IDs don't match colour.

Reply:
`pgm-maps/maps/race_drought/map.xml - 'team' attribute of 'spawn' element @ line 39: Missing team with ID '--team-black'`

I get these errors from the console, the XML fixed it so yeah lol